internal/ecrconsumer/image_values.go

```diff
@@ -0,0 +1,114 @@
+package ecrconsumer
+
+import (
+	"fmt"
+	"reflect"
+	"strings"
+
+	"ship-it/internal/helmrelease"
+
+	"github.com/google/go-github/github"
+)
+
+type GitCommands interface {
+	UpdateFile(msg string, branch string, path string, fileContent []byte) (*github.RepositoryContentResponse, error)
+	GetFile(branch string, path string) ([]byte, error)
+}
+
+type Image struct {
+	Registry   string
+	Repository string
+	Tag        string
+}
+
+func parseImage(repo string, tag string) (*Image, error) {
+	arr := strings.Split(repo, "/")
+
+	var registry string
+	var repository string
+	if len(arr) == 2 {
+		repository = arr[1]
+		registry = arr[0]
+	} else {
+		return nil, fmt.Errorf("repository field is invalid! (lacking either the full registry name or repository name)")
```

Comment:
Prefer to use `errors.New` instead of `fmt.Errorf` when there's no need for string formatting.

In this case, it would be helpful to include the malformed `repo` string, so we could use `fmt.Errorf` to add it to the error message.